src/nipple.d.ts

```diff
@@ -0,0 +1,217 @@
+declare interface ManagerOptions {
```

Comment:
Added most of the doc comments from the README, which is very helpful in a TypeScript aware editor.